addons/point_of_sale/static/src/js/ControlButtonsMixin.js

```diff
@@ -11,22 +11,80 @@ odoo.define('point_of_sale.ControlButtonsMixin', function (require) {
      * @param {Function} x superclass
      */
     const ControlButtonsMixin = (x) => {
+        let sorted = false;
```

Comment:
Why moving logic here (out of "addControlButton"), it seems more complex and not needed

Reply:
You want to remove the declaration of the control buttons from the manifest so I removed it. But to maintain the ordering, I have to do the ordering when every popup component is added, thus, ordering shouldn't be made in addControlButton.

Reply:
Okay, I get it, I'll try to find if there is a prettier way to order it

Reply:
Maybe we can sort the control buttons at setup of the Chrome component?

Reply:
Yes, you can move it there